zh/option/partial/axisPointer-common.md

```diff
@@ -265,7 +265,7 @@ label 距离轴的距离。
     prefix = "#" + ${prefix},
     defaultColor = "#555",
     defaultWidth = 1,
-    defaultType = 'solid'
+    defaultType = 'dashed'
```

Comment:
`defaultColor` looks wrong as well. And the same changes should be applied to the English documentation.